ai_serving/fasttext_clf/predict.py

```diff
@@ -0,0 +1,27 @@
+import sys
+
+from train import load_model
+from json_to_train import label_to_intent, prep_sentence
+
+
+def predict(sentences, model=None):
```

Comment:
I'd recommend make this a member function of the FastTextCLF class and then the script can be a simple wrapper of creating an instance of the class and calling its predict member function.